### DISCUSSION TITLE
Show search bar at the bottom of the screen instead of search button

### DISCUSSION THREAD

**Discussion by TobiPeterG** [osmandapp/OsmAnd#20922] [Ideas]
It would be more convenient if a search bar was shown at the bottom of the screen.
To the left and right of it could be buttons to e.g. switch the profile and for the settings.
Swiping it up could show recent searches/viewed POIs, while tapping on it could open a full-screen search window.

Swiping it down could hide it to allow a full-screen map view.

**Comment by DmitryAlexei** [comment]
Full screen mode can be activated in the profile settings. Please see https://osmand.net/docs/user/personal/profiles#other
